src/Mantasflowers.WebApi/Extensions/HttpHeadersExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Net.Http.Headers;
+
+namespace Mantasflowers.WebApi.Extensions
+{
+    public static class HttpHeadersExtensions
```

Comment:
We have `ResponseHeadersExtensions.cs`. Maybe these two files can be merged into one?